content/chapters/data/lab/content/process-memory.md

```diff
@@ -566,31 +567,24 @@ and let's build the two executable files:
 student@os:~/.../lab/support/copy$ make
 ```
 
-and run them:
+Run the `benchmark_cp.sh` script:
```

Comment:
Also mention that students should generate the input files first. You can use this makefile from the I/O lab: https://github.com/open-education-hub/operating-systems/blob/8ffedab662d2c2d20a6b0d75efe95d0f8365e47f/content/chapters/io/lab/support/file-mappings/Makefile

Reply:
The file is generated as the first step of this tutorial step with `./generate.sh` [1]

 [1] https://github.com/open-education-hub/operating-systems/pull/200/files/0801fedca2c02b8493748ff831faef5ffd039b45#diff-b84e4b0bb765638417bef0254489e3e46a817500e6ab65cf5e18751378d52726R561

Reply:
Oh, right. It should be fine. Sorry, I didn't see it.